site/docs/pages/wallet/types.mdx

```diff
@@ -74,6 +74,21 @@ export type WalletDropdownDisconnectReact = {
 };
 ```
 
+## `WalletDropdownFundLinkReact`
+
+```ts
+export type WalletDropdownFundLinkReact = {
+  className?: string; // Optional className override for the element
+  href: string;
+  icon?: ReactNode;
+  rel?: string;
+  openIn?: 'window' | 'tab'; // Whether to open the funding flow in a tab or a window
+  target?: string; // Where to open the target if `openIn` is set to tab
+  text?: string; // Optional text override
+  windowSize?: 's' | 'm' | 'l'; // Size of the popup window if `openIn` is set to window
```

Comment:
same here, is this a `popupSize` ?

Reply:
see above comment on usage of `window` instead of `popup`

Reply:
yeah `window` is a very special word in Javascript, not sure how I feel in using that.